.github/workflows/jepsen-single-instance-txm.yml

```diff
@@ -19,13 +16,15 @@ jobs:
 
     # image built by .gitlab.mk instructions and targets from .travis.mk
```

Comment:
.travis.mk is still in comment line
the same are [here](https://github.com/tarantool/tarantool/pull/7354/files#diff-451818a5d6879093b617bbec954d15428ed6825a568ad3b60dc58434437611ccR12) and [here](https://github.com/tarantool/tarantool/pull/7354/files#diff-17526634afb86c17df0cb865811a603a0601b179c533e8817e819650bad0858bR17)
I suppose you should do like `grep -Rni '.travis.mk' .` before merge

Reply:
Looks like there is no need to fix .gitlab.mk because all this stuff will be removed in https://github.com/tarantool/tarantool/pull/7406.

Reply:
In other files we should fix it.

Reply:
Fixed.

```bash
$ grep --exclude '*/.git/*' -Rni '.travis.mk' .
./.gitlab.mk:2:TRAVIS_MAKE:=${MAKE} -f .travis.mk
./.gitlab.mk:4:# Pass *_no_deps goals to .travis.mk.
./.gitlab.mk:25:# When some of deps_* goals in .travis.mk are updated.
./.gitlab.mk:35:COPY .travis.mk .
./.gitlab.mk:36:RUN make -f .travis.mk deps_debian
./.gitlab.mk:42:COPY .travis.mk .
./.gitlab.mk:43:RUN make APT_EXTRA_FLAGS="--allow-releaseinfo-change-version --allow-releaseinfo-change-suite" -f .travis.mk deps_buster_clang_8 deps_buster_clang_11
./.gitlab.mk:51:TRAVIS_CI_MD5SUM:=$(firstword $(shell md5sum .travis.mk))
./.github/workflows/luajit-integration.yml:85:          # section, the branching is moved to .travis.mk and
./.github/workflows/luajit-integration.yml:88:          make -f .travis.mk "luajit_$(uname -s)_${ARCH}_test"
./.travis.mk:47:# settings back (e.g. when running tests via .travis.mk on the
```